openquake/calculators/postproc/compute_rtgm.py

```diff
@@ -750,17 +762,19 @@ def main(dstore, csm):
         dstore.create_df('rtgm', pd.concat(rtgm_dfs))
 
     plot_sites(dstore, update_dstore=True)
-    if rtgm_dfs and N == 1:  # and not warnings[sid]:
+    if rtgm_dfs and N == 1:
         sid = 0
-        if not warnings[sid].startswith(('Zero hazard', 'Very low hazard')):
+        if not warnings:
```

Comment:
Please @kejohnso and @ManuelaVillani check also if the creation of these plots is still made under the correct conditions